themes/half-life/half-life.theme.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+OSH_THEME_GIT_PROMPT_DIRTY="✗"
+OSH_THEME_GIT_PROMPT_CLEAN="✓"
+
+# Nicely formatted terminal prompt
+ps_scm_prompt() {
+  CHAR=$(scm_char)
+  if [ $CHAR = $SCM_NONE_CHAR ]
+    then
+      return
+    else
+      echo " on ${blue}$(git_current_branch)$(parse_git_dirty)${normal} "
+  fi
+}
+
+function prompt_command() {
```

Comment:
It is better to make these two functions consistent.

```suggestion
prompt_command() {
```